packages/core/src/sass/app/modules/set-residence.scss

```diff
@@ -21,6 +21,10 @@
         margin: 0 0 2.4rem;
         max-width: 90%;
     }
+    &__button_wrapper {
+        margin-top: 1.6rem;
+        text-align: right;
```

Comment:
can we pass this as prop to Text

Reply:
This class was added to keep the submit button to the right side. `children` will contain `Text` and `Button` components coming from the parent component. could you please specify to which Text component we'll be able to pass it as prop?
`<div className={`${class_prefix}__button_wrapper`}>{children}</div>`

Reply:
i have used flex box styling instead of text-align for the wrapper div.